source/tagged-union.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+Utility type to create a tagged union with given tag
```

Comment:
The first sentence in this file should be the same as the readme.

Reply:
Good point! I'll fixed this :)